source/dubregistry/api.d

```diff
@@ -95,6 +95,7 @@ interface IPackages {
 	@path(":name/:version/info")
 	Json getInfo(string _name, string _version, bool minimize = false);
 
+	@path(":name/:version/infos")
 	Json[string] getInfos(string[] packages, bool include_dependencies = false, bool minimize = false);
```

Comment:
Shouldn't there be `_name` and `_version` parameters in the signature here? Otherwise the previous endpoint ("/infos") seems to be fine.

Reply:
(Previous one is working fine: http://code.dlang.org/api/packages/infos?packages=[%22dub%22])